### PR TITLE
Add Project DDP-Set up a monitoring system

### PR DESCRIPTION
**Pull request by jayanth-kumar-morem**

### CODE REVIEW

File: docs/2023/DDP/Set up a monitoring system/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+slug: /2023/DDP/Set up a monitoring system
+---
+
+
+# Author
+Jayanth Kumar
+
+# Contact Info
+- [Email](mailto:jayanthkumar1903@gmail.com)
+[Linked In](https://www.linkedin.com/in/jayanth-kumar-14a4811b5/)
```
Comment (Apoorvgarg-creator) [inline docs/2023/DDP/Set up a monitoring system/index.md:13]:
```suggestion
- [LinkedIn](https://www.linkedin.com/in/jayanth-kumar-14a4811b5/)
```

**Review by Apoorvgarg-creator — APPROVED** [review]

File: docs/2023/DDP/Set up a monitoring system/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+
+# Author
+Jayanth Kumar
+
+# Contact Info
+- [Email](mailto:jayanthkumar1903@gmail.com)
+- [LinkedIn](https://www.linkedin.com/in/jayanth-kumar-14a4811b5/)
+[GitHub](github.com/jayanth-kumar-morem)
```
Comment (Shruti3004) [inline docs/2023/DDP/Set up a monitoring system/index.md:14]: - [GitHub](https://github.com/jayanth-kumar-morem)

**Review by Shruti3004 — CHANGES_REQUESTED** [review]

File: docs/2023/DDP/Set up a monitoring system/index.md (hunk 2 above)
Comment (Shruti3004) [reply docs/2023/DDP/Set up a monitoring system/index.md:14]: Write https:// in front of your GitHub username

Comment (claddyk) [inline docs/2023/DDP/Set up a monitoring system/index.md:14]:
```suggestion
- [Email](mailto:jayanthkumar1903@gmail.com)
- [LinkedIn](https://www.linkedin.com/in/jayanth-kumar-14a4811b5/)
- [GitHub](https://github.com/jayanth-kumar-morem)
```

**Review by claddyk — CHANGES_REQUESTED** [review]

**Pull request merged**